update the timber fluentd docs to indicate that hostname is required

notatoad opened the pull request:
the fluent-plugin-timber readme says the hostname is mandatory, and my installation was failing without it which seems to support that claim

binarylogic commented:
@LucioFranco notice your Elixir changes do not follow the same pattern as Ruby: https://docs.timber.io/languages/ruby/installation/

We need to decide on one. My vote is how Ruby is structured. The manual installation is a more laborious process of what the automatic installation is doing. They are one in the same.

LucioFranco commented:
@binarylogic well the manual page doesn't exist? and it seems its actually under the same URL. I think having two pages like in the Elixir page makes more sense and is clear at least.

notatoad committed:
Update README.md

the fluent-plugin-timber readme says the hostname is mandatory, and my installation was failing without it which seems to support that claim
